README

```diff
@@ -698,6 +698,11 @@ MPI Collectives
   collectives, copies the data to staging buffers if GPU buffers, then
   calls underlying collectives to do the work.
 
+- The "ucc" coll component serves as integration layer for Unified
+  Collective Communication (UCC) library: https://github.com/openucx/ucc.
+  In order to build OMPI with UCC support pass "--with-ucc=<path_to_ucc>"
+  as one of the configuration parameters.
```

Comment:
Please remove this sentence and add --with-ucc with the rest of the NETWORKING SUPPORT OPTIONS section, later.